src/styles/utils/spacing.ts

```diff
@@ -131,6 +131,10 @@ export default {
         marginRight: 0,
     },
 
+    mrHalf: {
+        marginRight: 2,
+    },
+
```

Comment:
This does not fit into design language, let's not create a common style for this in that case. You can direclty apply the margin where needed.


Our design follow: (x * 4) pattern